content/4_Gold/DSU.mdx

```diff
@@ -172,20 +178,46 @@ class DisjointSets:
 		y_root = self.find(y)
 		if x_root == y_root:
 			return False
-		
-		if self.sizes[x_root] < self.sizes(y_root):
+
+		if self.sizes[x_root] < self.sizes[y_root]:
 			x_root, y_root = y_root, x_root
 		self.parents[y_root] = x_root
 		self.sizes[x_root] += self.sizes[y_root]
 		return True
+
+	# returns whether two nodes are in the same connected component
+	def connected(self, x: int, y: int) -> bool:
+		return self.find(x) == self.find(y)
 ```
 
 </PySection>
 </LanguageSection>
 
-## Focus Problem Solution
+## Solution - Focus Problem
 
-<FocusProblem problem="sample" />
```

Comment:
was this supposed to be deleted?

Reply:
yeah, the focus problem showed up twice, which was kind of confusing